src/SFA.DAS.QnA.Api/Controllers/ApplicationDataController.cs

```diff
@@ -58,7 +57,7 @@ public async Task<ActionResult<string>> GetQuestionTagData(Guid applicationId, s
 
             if (!applicationDataResponse.Success) return NotFound(new NotFoundError(applicationDataResponse.Message));
 
-            return applicationDataResponse?.Value;
+            return applicationDataResponse?.Value.ToString();
```

Comment:
The ? is not required (I know you didn't write it) as applicationDataResponse is never null here - if it was null it would already have crashed on the line above but the handler does not return null - and the ToString() is not required as the Value is already a string so this should just be 

return applicationDataResponse.Value;